core/spec/models/spree/stock/splitter/weight_spec.rb

```diff
@@ -6,12 +6,15 @@ module Splitter
       describe Weight, :type => :model do
         let(:packer) { build(:stock_packer) }
         let(:variant) { build(:base_variant, :weight => 100) }
+        let(:heavy_variant) { build(:base_variant, :weight => 49) }
```

Comment:
Could you switch the names of these around it bugs my ocd that the heavy variant is the lighter one, and would be good to switch the hash syntax as well.